pom.xml

```diff
@@ -103,6 +103,12 @@
             <scope>test</scope>
         </dependency>
         <dependency>
+            <groupId>org.jenkins-ci.plugins.workflow</groupId>
+            <artifactId>workflow-step-api</artifactId>
+            <version>1.4</version>
```

Comment:
imho we should target 1.11 or workflow-plugin, but if this works then let's keep it like that.